quickwit-cli/src/main.rs

```diff
@@ -276,17 +282,43 @@ fn about_text() -> String {
     about_text
 }
 
+/// Parse duration with unit.
+/// examples: 1s 2m 3h 5d
+pub fn parse_duration_with_unit(duration: &str) -> anyhow::Result<Duration> {
```

Comment:
no need to make the grace period configurable, so we can remove this. Let's keep the snippet and the test though. I am sure it will be useful

Reply:
Marking as resovled as apparently it was requested by Adrien and Frnacois